components/announcement_bar/announcement_bar.jsx

```diff
@@ -246,13 +242,6 @@ export default class AnnouncementBar extends React.PureComponent {
                     defaultMessage='Preview Mode: Email notifications have not been configured'
                 />
             );
-        } else if (message === ErrorBarTypes.APIV3_ENABLED) {
-            message = (
-                <FormattedHTMLMessage
-                    id={ErrorBarTypes.APIV3_ENABLED}
-                    defaultMessage='API version 3 is deprecated and scheduled for removal. <a href="https://api.mattermost.com/#tag/APIv3-Deprecation" target="_blank">Learn how to migrate to APIv4</a>.'
-                />
-            );
```

Comment:
This remove the API  version 3 deprecation warning, I prefer that @jwilander confirm that want this change on master.

Reply:
Yes it's fine going into master